fenix/app/src/test/java/org/mozilla/fenix/tabstray/DefaultTabsTrayControllerTest.kt

```diff
@@ -865,6 +865,34 @@ class DefaultTabsTrayControllerTest {
         verify { controller.handleNavigateToBrowser() }
     }
 
+    @Test
+    fun `GIVEN a tab is selected WHEN handleTabDeletion is called on last private tab THEN that private tab is removed and an undo snackbar is shown and original tab is still displayed`() {
```

Comment:
```suggestion
    fun `GIVEN a normal tab is selected WHEN the last private tab is deleted THEN that private tab is removed and an undo snackbar is shown and original normal tab is still displayed`() {
```

We try to avoid referencing the function names in the test names, since those can get later refactored away but also it keeps the GIVEN WHEN THEN more of a sequence of contextual points and actions rather than a result of code being called.

Reply:
gotcha, made the change to the title. thank you!